src/averageAges.js

```diff
@@ -37,7 +51,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => {
+    return (person.sex === 'f')
+      && (withChildren ? people.some(child => child.mother === person.name)
+        : true);
```

Comment:
краще переносити отак, якшо довге, решту тернарів теж подивися
```suggestion
      && (withChildren
        ? people.some(child => child.mother === person.name)
        : true);
```